framework/resources/Advanced/runtimefilter/tpch_sf100/parquet/broadcast_join_double.q

```diff
@@ -0,0 +1,16 @@
+alter session set `exec.hashjoin.enable.runtime_filter` = true;
+alter session set `exec.hashjoin.runtime_filter.max.waiting.time` = 10000;
+
+select
+  count(*) as row_count
+from
+  orders o
+where
+  o.o_totalprice = (
```

Comment:
This is interesting. 
I see the planner figured that there is only 1 row in the join. Is there a way to confirm from a query profile on the number of rows emitted by the `01-05` (SVR)? 
If the filter applies correctly, there should be only 1 row emitted, since the wait time is long.

Reply:
Yes, the build side emits 1 row to hash join operator. Did you want this check to be a part of the verification?

Reply:
Yes, the build side emits 1 row to hash join operator. Did you want this check to be a part of the verification?

Reply:
No. i think we can skip the check.